nfv_tempest_plugin/tests/scenario/test_nfv_advanced_usecases.py

```diff
@@ -150,9 +150,15 @@ def test_numa_aware_vswitch(self, test='numa_aware_vswitch'):
 
         if (not CONF.nfv_plugin_options.target_hypervisor
                 and CONF.nfv_plugin_options.run_live_migration):
-            migrate_kw_args = {'block_migration': True, 'force': True}
+            migrate_kw_args = {
+                'block_migration':
+                CONF.nfv_plugin_options.live_migration_block,
+                'force': True
+            }
             # on 16.X microversion is 2.72
-            if float(CONF.compute.min_microversion) > 2.67:
+            if (float(CONF.compute.min_microversion) > 2.67
+                and CONF.nfv_plugin_options.nfv_plugin_options.
+                live_migration_block):
```

Comment:
still problematic code, IMO. Please take a look at lines 151-152. I would do in the same style, i.e.
```
if (float(CONF.compute.min_microversion) > 2.67
        and CONF.nfv_plugin_options.live_migration_block):
```
Also my example removes the duplicate of nfv_plugin_options in the CONF value, otherwise it won't be found.
BUT, according to the condition in line 152 the CONF.nfv_plugin_options.live_migration_block is expected to be True anyway in line 160, otherwise the program won't be able to reach the line 160. So, probably this 'and' condition from the line 160 is redundant. IIUC, if CONF.nfv_plugin_options.run_live_migration is 'false' all code starting from line 153 till the end of the test will be skipped.

Reply:
please can you share the output of your flake8 execution, I don't see any error in my execution:
```
(venv) rdiazcam@rdiazcam-thinkpadt14sgen1:~/Git/github.com/NFV18/nfv-tempest-plugin$ flake8 nfv_tempest_plugin/tests/scenario/test_nfv_advanced_usecases.py
(venv) rdiazcam@rdiazcam-thinkpadt14sgen1:~/Git/github.com/NFV18/nfv-tempest-plugin$ 
```

Reply:
flake8 works for me, sorry for disturbing. though IMO not readable well but won't bother with this anymore :)
I will let someone more familiar with the code to review. :)

Reply:
IMHO my first version was more readable but it needs to be like that if we don't want flake8 to complain.